<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>420</width>
     <height>490</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>38</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>38</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>38</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext3">
   <property name="geometry">
    <rect>
     <x>786</x>
     <y>168</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext2">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>168</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext1">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>168</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>188</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext4">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>318</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>188</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_6">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>188</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>358</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>358</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow_9">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>358</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext7">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>488</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext8">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>488</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext9">
   <property name="geometry">
    <rect>
     <x>786</x>
     <y>488</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext5">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>318</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeltext6">
   <property name="geometry">
    <rect>
     <x>786</x>
     <y>318</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelId">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>600</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>LastTime</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSex">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditId">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>528</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTime">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>598</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>528</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSex">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>598</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>600</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
